include/deal.II/base/vectorization.h

```diff
@@ -245,18 +245,17 @@ class VectorizedArrayIterator
  *
  * @author Peter Munch, 2019
  */
-template <typename T>
+template <typename T, std::size_t width>
 class VectorizedArrayBase
 {
 public:
   /**
-   * Return the number of elements in the array stored in the variable
-   * n_array_elements of VectorizedArray.
+   * Return the number of elements in the array.
    */
   static constexpr std::size_t
   size()
   {
-    return T::n_array_elements;
+    return width;
```

Comment:
@bangerth Answering https://github.com/dealii/dealii/pull/9688#discussion_r395721182: this is the reason. We cannot access `n_array_elements` anymore.